modules/L10_system_emulation/run_service.sh

```diff
@@ -27,7 +27,7 @@ if ("${FIRMAE_ETC}"); then
       BINARY_NAME=$("${BUSYBOX}" basename "${BINARY_NAME}")
       if ( ! ("${BUSYBOX}" ps | "${BUSYBOX}" grep -v grep | "${BUSYBOX}" grep -sqi "${BINARY_NAME}") ); then
         "${BUSYBOX}" echo "[*] Starting $BINARY_NAME service ..."
-        "${BINARY}" &
+        ${BINARY} &
```

Comment:
this on purpose

Reply:
yes ... otherwise the parameters of some commands are not working